rosidl_typesupport_introspection_c/cmake/rosidl_typesupport_introspection_c_generate_interfaces.cmake

```diff
@@ -129,12 +144,15 @@ add_dependencies(
 )
 
 if(NOT rosidl_generate_interfaces_SKIP_INSTALL)
-  # TODO service headers
   install(
-    FILES ${_generated_header_files}
+    FILES ${_generated_msg_header_files}
     DESTINATION "include/${PROJECT_NAME}/msg"
   )
   install(
+    FILES ${_generated_srv_header_files}
+    DESTINATION "include/${PROJECT_NAME}/srv"
+  )
```

Comment:
Both install commands need to be wrapped in a conditional if the variable is not empty.